src/Shouldly.Tests/Shouldly.Tests.csproj

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net472;</TargetFrameworks>
-    <LangVersion>latest</LangVersion>
```

Comment:
Moved to Directory.Build.props so that it could be next to `<Nullable>enable</Nullable>`